game/src/views/game.vue

```diff
@@ -0,0 +1,79 @@
+<script setup lang="ts">
+import { ref,computed } from 'vue'; 
+
+const showMessage = ref(false);
+const showMessageWrong = ref(false);
+const showMessageWrong_2 = ref(false);
+const showMessageWrong_3 = ref(false);
+
+const handleButtonClick = () => {
+  showMessage.value = true;
+};
+
+const handleButtonClickWrong_1 = () => {
+  showMessageWrong.value = true;
+};
+
+const handleButtonClickWrong_2 = () => {
+  showMessageWrong_2.value = true;
+};
+
+const handleButtonClickWrong_3 = () => {
+  showMessageWrong_3.value = true;
+};
+
+const goldfinchColor = computed(() => (showMessage.value ? 'success' : 'primary'));
+const bluebirdColor = computed(() => (showMessageWrong.value ? 'error' : 'primary'));
+const siskinColor = computed(() => (showMessageWrong_2.value ? 'error' : 'primary'));
+const meadowlarkColor = computed(() => (showMessageWrong_3.value ? 'error' : 'primary'));
+
+
+</script>
+
+<template>
+  <div class="game">
+    <v-card>
+      <v-card-title>
+        <h2>What is the state bird of WA state?</h2>
```

Comment:
```suggestion
        What is the state bird of WA state?
```
No need to wrap your card title inside an h2 tag. If the v-card-title text isn't large enough for you, you can use the typography classes to customize it further!
https://vuetifyjs.com/en/styles/text-and-typography/